.gitignore

```diff
@@ -2,6 +2,7 @@
 /openshift.local.*
 /.project
 /.vagrant
+/assets/nbproject
```

Comment:
what is this for?  if its something only you are going to have locally, then lets not check this in